src/app/qgsdecorationcopyrightdialog.cpp

```diff
@@ -70,16 +70,8 @@ QgsDecorationCopyrightDialog::QgsDecorationCopyrightDialog( QgsDecorationCopyrig
   wgtUnitSelection->setUnits( QgsUnitTypes::RenderUnitList() << QgsUnitTypes::RenderMillimeters << QgsUnitTypes::RenderPercentage << QgsUnitTypes::RenderPixels );
   wgtUnitSelection->setUnit( mDeco.mMarginUnit );
 
-  // color
-  pbnColorChooser->setAllowOpacity( true );
-  pbnColorChooser->setColor( mDeco.mColor );
-  pbnColorChooser->setContext( QStringLiteral( "gui" ) );
-  pbnColorChooser->setColorDialogTitle( tr( "Select Text color" ) );
-
-  QTextCursor cursor = txtCopyrightText->textCursor();
-  txtCopyrightText->selectAll();
-  txtCopyrightText->setTextColor( mDeco.mColor );
-  txtCopyrightText->setTextCursor( cursor );
+  // font settings
+  mButtonFontStyle->setTextFormat( mDeco.textFormat() );
```

Comment:
Can you also:

- set the font button dialog title
- set the font button map canvas (i figure why not... it'll unlock some extra canvas related functionality in the dialog, which may not be terribly useful here, but at the cost of a single line of code may come in useful for 1 or 2 users!)